adoc/iv-Application/SA.adoc

```diff
@@ -95,8 +107,15 @@ https://console.aws.amazon.com/cloudshell/
 
 endif::[]
 
+ifeval::[ "{cloud}" == "GCP" ]
+
+In addition, GCP provides an easy to access https://shell.cloud.google.com/[web based command line shell] where Terraform is already pre-installed.
```

Comment:
GCP should be a declared at some point before it got used - not sure if it is done.
Other idea would be to use a variable in such cases which we can change on the fly

Reply:
Good point. I added `{gcloud}` variable to the document the `SA_vars.adoc` file.